sources/packages/api/src/route-controllers/program-info-request/models/program-info-request.dto.ts

```diff
@@ -1,15 +1,22 @@
-import { IsNotEmpty, IsInt, Min, IsOptional } from "class-validator";
-import { SaveOfferingDTO } from "../../education-program-offering/models/education-program-offering.dto";
+import {
+  IsNotEmpty,
+  IsInt,
+  Min,
+  IsOptional,
+  IsPositive,
+} from "class-validator";
+import { SaveOfferingDTO } from "src/route-controllers/education-program-offering/models/education-program-offering.dto";
 import { ProgramInfoStatus } from "../../../database/entities";
 import { OfferingIntensity } from "../../../database/entities/offering-intensity.type";
 
-export interface CompleteProgramInfoRequestDto extends SaveOfferingDTO {
-  selectedProgram?: number;
-  selectedOffering?: number;
+export class CompleteProgramInfoRequestAPIInDTO {
+  @IsPositive()
+  selectedOffering: number;
 }
 
-export interface GetProgramInfoRequestDto
-  extends CompleteProgramInfoRequestDto {
+export interface GetProgramInfoRequestDto extends SaveOfferingDTO {
+  selectedOffering?: number;
+  selectedProgram?: number;
```

Comment:
I had to create these 2 attributes here because they are used in the method getProgramInfoRequest.

<img width="526" alt="image" src="https://user-images.githubusercontent.com/72251620/166308134-98f825e3-94a4-48c9-aa2b-a32dc56f69c0.png">